### PR TITLE
Feature: Add latest_report_status to puppetdb backend provider #22167

### PR DESCRIPTION
**Pull request by negast**
This pull request is made for feature #22167
I just added the latest report status to the tags. because it is usefull info about the nodes

### CODE REVIEW

**Review by nuritizra — APPROVED** [review]

**Comment by negast** [comment]
I added some stuff to pase the check. I don't really know how to fix stuff to pass the remaining checks? For the DCO check should I just create the fork again or can I rebase with the latest commit?

File: .changeset/wet-lions-crash.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-puppetdb': patch
+---
+
+Added latest_report_status parameter from the puppetdb node api and added it as a tag to the nodes. The status is valuable information as it displays which nodes are compliant to your configuration and which ones are failing are making changes.
```
Comment (Rugvip) [inline .changeset/wet-lions-crash.md:5]:
```suggestion
Added `latest_report_status` parameter from the PuppetDB node api and added it as a tag to the nodes. The status is valuable information as it displays which nodes are compliant to your configuration and which ones are failing are making changes.
```

**Review by Rugvip — CHANGES_REQUESTED** [review]
Nice thank you! 👍 

Just some changeset nits.

The E2E tests should not be fixed so you don't need to worry about those. We'll need DCO to be able to merge this though, see the [check details](https://github.com/backstage/backstage/pull/22195/checks?check_run_id=20493142819) for more info on how to get that resolved.

**Comment by negast** [comment]
I'm just gonna remake the fork I dont know how to fix the previous commit DCO

**Pull request closed without merging**